paf-mvp-core-js/src/crypto/verifier.ts

```diff
@@ -24,7 +34,12 @@ export class Verifier<T> {
     const signature = this.definition.getSignature(signedData);
     const toVerify = this.definition.getInputString(signedData);
 
-    return publicKey.verify(toVerify, signature);
+    const result = publicKey.verify(toVerify, signature);
+
+    if (result) this.logger.Debug('Verified', signedData);
+    else this.logger.Error('Verification failed for', signedData);
```

Comment:
Same remark here. I am wondering for example if a user deny GDPR consent, we may don't want to log the identifier.